oam/install_scripts/run.sh

```diff
@@ -26,9 +29,9 @@ while getopts "vs:t:h" flag; do
 	esac
 done
 
-shift $((OPTIND - 1))
+shift $((OPTIND-1))
 
-exename="$1"
+exename="$@"
```

Comment:
I'd have to run this to make sure I've got the alignment right in my head.  At first glance this would expand, I think, to the target executable + all of its args.  Is that what we want here?